### PR TITLE
[DO-80] adding efs mount to ecs cluster creation

### PR DESCRIPTION
**Pull request by simplyadrian**

### CODE REVIEW

**Comment by simplyadrian** [comment]
@notyost @sapiensantiquus

File: tests/vars/test_aws_info.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-ecs_host_vpc: "YOUR-VPC-ID"
-ecs_subnets: "YOUR-SUBNET-ID,MOAR-SUBNET-ID"
-ecs_sg_ids: "YOUR-SG-ID,MOAR-SG-ID"
-ecs_host_keypair_name: "YOUR-KEY-PAIR-NAME"
-ecs_efs_id: "YOUR-EFS-FILESYSTEM-ID"
+ecs_host_vpc: "vpc-fa3aca9e"
+ecs_subnets: "subnet-27371d7e,subnet-2d29365a"
```
Comment (sapiensantiquus) [inline tests/vars/test_aws_info.yml]: Did you mean to commit these changes?

File: templates/ecs.json.j2 [hunk 2]
```diff
@@ -135,9 +136,8 @@
         "SecurityGroups" : { "Ref" : "SecurityGroupIDs" },
         "UserData"       : { "Fn::Base64" : { "Fn::Join" : ["", [
              "#!/bin/bash -xe\n",
-             "yum install -y aws-cfn-bootstrap\n",
-             "yum install -y nfs-utils\n",
-             "echo "$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone).{ "Ref": "FileSystem" }.efs.{ "Ref" : "AWS::Region" }.amazonaws.com:/    /mnt/efs  nfs4    defaults" >> /etc/fstab\n",
+             "yum install -y nfs-utils aws-cfn-bootstrap\n",
+             "echo \\\"$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone).{ \\\"Ref\\\": \\\"FileSystem\\\" }.efs.{ \\\"Ref\\\" : \\\"AWS::Region\\\" }.amazonaws.com:/    /mnt/efs  nfs4    defaults\\\" >> /etc/fstab\n",
              "mkdir /mnt/efs\n",
```
Comment (sapiensantiquus) [inline templates/ecs.json.j2]: Nice

**Comment by sapiensantiquus** [comment]
Looks good 👍  maybe remove stuff specific to our account though?

**Comment by notyost** [comment]
LGTM 🌴

**Pull request merged**